### ISSUE TITLE
List items should have role 'listitem'

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#90876]
Currently the list sets the role of each item to be `treeitem` [here](https://github.com/microsoft/vscode/blob/8451f8bd913914aac17b6b3273b6ed0db137a217/src/vs/base/browser/ui/list/listView.ts#L552)
This should only be done for the Tree, not for the list. List should use `listitem`. More details [here](https://github.com/nvaccess/nvda/issues/10777#issuecomment-586909490)

Tree that actually uses the list should probably pass the role `treeitem` as the getRole to the list it extends from.
@joaomoreno if you are busy let me know and I can look into this.

**Comment by isidorn** [comment]
We should also verify this nicely works with NVDA, since it is part of aria 1.1 spec.

**Comment by joaomoreno** [comment]
Sure, PR would be great.

**Comment by isidorn** [comment]
Sure. Assigning to this milestone and if I do not find time might get pushed out.

**Comment by isidorn** [comment]
To verify
1. Windows, turn on NVDA
2. Open suggest box -> make sure that the suggestions do not have "Level 1" read ath the end
3. Open explorer -> make sure the items have appropriate Level read at the end
